src/schema/contentful/phoenix.js

```diff
@@ -118,6 +154,25 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type ContentBlock implements Block {
+    "The internal-facing title for this link block."
+    internalTitle: String!
+    "An small decorated title positioned over the general title"
+    superTitle: String!
+    "The user-facing title of the block"
+    title: String!
+    "A subtitle that is never displayed"
+    subTitle: String!
+    "Text to display regarding content subject"
+    content: String
+    "Image relevant to the content"
+    image: Asset
+    "The alignment of the image"
+    imageAlignment: String!
```

Comment:
This field isn't required in Contentful
```suggestion
    imageAlignment: String
```